DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/FamilyInstanceConversionToSpeckle.cs

```diff
@@ -1,32 +1,32 @@
-using Autodesk.Revit.DB;
-using Autodesk.Revit.DB.Structure;
+using Speckle.Converters.Common;
 using Speckle.Converters.Common.Objects;
-using Speckle.Converters.RevitShared.Extensions;
 using Speckle.Converters.RevitShared.Helpers;
 using Speckle.Core.Models;
+using Speckle.Converters.RevitShared.Extensions;
 
 namespace Speckle.Converters.RevitShared.ToSpeckle;
 
 // POC: bin for now? This is also a parent child relationship and may need a pattern for this
 // so we don't end up with some god FamilyInstanceConversionToSpeckle converter
+[NameAndRankValue(nameof(DB.FamilyInstance), NameAndRankValueAttribute.SPECKLE_DEFAULT_RANK)]
 public sealed class FamilyInstanceConversionToSpeckle : BaseConversionToSpeckle<DB.FamilyInstance, Base>
 {
-  private readonly IRawConversion<DB.Element> _elementConverter;
+  private readonly IRawConversion<DB.Element, SOBR.RevitElement> _elementConverter;
   private readonly IRawConversion<DB.FamilyInstance, SOBR.RevitBeam> _beamConverter;
   private readonly IRawConversion<DB.FamilyInstance, SOBR.RevitColumn> _columnConverter;
 
   public FamilyInstanceConversionToSpeckle(
-    IRawConversion<Element> elementConverter,
-    IRawConversion<FamilyInstance, SOBR.RevitBeam> beamConverter,
-    IRawConversion<FamilyInstance, SOBR.RevitColumn> columnConverter
+    IRawConversion<DB.Element, SOBR.RevitElement> elementConverter,
+    IRawConversion<DB.FamilyInstance, SOBR.RevitBeam> beamConverter,
+    IRawConversion<DB.FamilyInstance, SOBR.RevitColumn> columnConverter
   )
   {
     _elementConverter = elementConverter;
     _beamConverter = beamConverter;
     _columnConverter = columnConverter;
   }
 
-  public override Base RawConvert(FamilyInstance target)
+  public override Base RawConvert(DB.FamilyInstance target)
```

Comment:
So I suppose this is where having named IRawConversions would be handy and avoid the ever growing if statements/switch

Reply:
Yep! I think this is the next thing we should try to figure out in parallel, but for now I'm quite happy to do this bc it's simple, just as long as we flag them as POC